src/www/system_certmanager.php

```diff
@@ -864,6 +864,7 @@ function internalca_change() {
                     <select name="cert_type">
                         <option value="usr_cert" <?=$pconfig['cert_type'] == 'usr_cert' ? 'selected="selected"' : '';?>> <?=gettext("Client Certificate");?> </option>
                         <option value="server_cert" <?=$pconfig['cert_type'] == 'server_cert' ? 'selected="selected"' : '';?>> <?=gettext("Server Certificate");?> </option>
+                        <option value="combined_server_client" <?=$pconfig['cert_type'] == 'combined_server_client' ? 'selected="selected"' : '';?>> <?=gettext("Combined Client/Server Certificate");?> </option>
```

Comment:
```suggestion
                        <option value="combined_server_client" <?=$pconfig['cert_type'] == 'combined_server_client' ? 'selected="selected"' : '';?>> <?=gettext("Client/Server Certificate");?> </option>
```

Reply:
Are you sure this is clear enough to our average user?

Reply:
Looking at the drop down choices all at once, yes. "Combined" is a kind of doubling.

Reply:
the shorter the text the clearer the meaning. factor in translators who will "interpret" the meaning of the string. just want to point out best practice so deviating from it for a valid reason is a conscious decision and not just a gut feeling. your call, I don't mind either way.

Reply:
Ah, I see. Well, I think the original text is also clearer to translators (especially if they are not fully aware of the context). If you are okay with keeping the original text, then I'd like to keep it.

Reply:
Okay, sure